lib/ui/views/match/years/match-breakdown-2021.dart

```diff
@@ -6,122 +6,144 @@ import '../../../widgets/match-breakdown-row.dart';
 import 'package:toa_flutter/models/game-specifics/ultimategoal-match-details.dart';
 
 class MatchBreakdown2021 {
-  static List<Widget> getRows(Match match, BuildContext context) {
+  static List<Widget> getRows(Match match, BuildContext context, bool isRemote) {
     TOALocalizations local = TOALocalizations.of(context);
     UltimateGoalMatchDetails details = match.gameData;
```

Comment:
It makes more sense to check how many teams participating in the match.
```suggestion
    UltimateGoalMatchDetails details = match.gameData;
    bool isRemote =  match.participants.length == 1;
```